redash/query_runner/athena.py

```diff
@@ -71,7 +71,7 @@ def configuration_schema(cls):
                     "title": "Schema Name",
                     "default": "default",
                 },
-                "glue": {"type": "boolean", "title": "Use Glue Data Catalog"},
+                "schema_from_api": {"type": "boolean", "title": "Use Athena API for get_schema"},
```

Comment:
I wonder if we need this setting at all, i.e. is there a reason for someone to prefer loading schema with the SQL commands? One benefit of the SQL is that it will load everything in one go instead of iterating over multiple API calls. But I'm not sure if this is an issue.